optimade/server/entry_collections.py

```diff
@@ -35,6 +35,10 @@ def __contains__(self, entry):
     def get_attribute_fields(self) -> set:
         schema = self.resource_cls.schema()
         attributes = schema["properties"]["attributes"]
+        if "allOf" in attributes:
+            allOf = attributes.pop("allOf")
+            for dict_ in allOf:
+                attributes.update(dict_)
```

Comment:
This is new syntax right? How does one use it?

Reply:
I am not sure. I just saw it in the response and had to add in the extra logic. I will see if I can find some proper reference and documentation on this.

Reply:
I have found the [JSON-Schema Core](https://json-schema.org/draft/2019-09/json-schema-core.html), specifically [this section](https://json-schema.org/draft/2019-09/json-schema-core.html#rfc.section.9.2.1.1).